nav2_rviz_plugins/src/goal_tool.cpp

```diff
@@ -47,11 +47,13 @@ void
 GoalTool::onPoseSet(double x, double y, double theta)
 {
   std::string fixed_frame = context_->getFixedFrame().toStdString();
-
-  navigation_dialog_->startNavigation(x, y, theta, fixed_frame);
-  navigation_dialog_->show();
-  navigation_dialog_->raise();
-  navigation_dialog_->activateWindow();
+  bool start_navigation_result = true;
```

Comment:
This section you could re-write more elegantly as:

```
 if (navigation_dialog_->startNavigation(x, y, theta, fixed_frame)) {
```

You don't need to store the variable `start_navigation_result` and the `!= false` check is redundant for the boolean. 

:-)